src/reanimated2/PlatformChecker.ts

```diff
@@ -16,10 +16,14 @@ export function isAndroid(): boolean {
   return Platform.OS === 'android';
 }
 
+export function isMacos(): boolean {
```

Comment:
```suggestion
export function isMacOS(): boolean {
```

Reply:
Apple's capitalization scheme of "macOS" really doesn't play well with code capitalization schemes :P. I think `isMacos` is probably fine